packages/sources/alpine/src/endpoint/lastblock.ts

```diff
@@ -0,0 +1,33 @@
+import { Validator, util } from '@chainlink/ea-bootstrap'
+import { Config, ExecuteWithConfig, InputParameters } from '@chainlink/types'
+import { ethers } from 'ethers'
+import stagingAbi from '../abi/stagingContract.json'
+
+export const supportedEndpoints = ['lastblock']
+
+export const inputParameters: InputParameters = {
+  stagingAddress: true,
+}
+
+export const execute: ExecuteWithConfig<Config> = async (request, _, config) => {
+  const validator = new Validator(request, inputParameters)
+  if (validator.error) throw validator.error
+
+  const jobRunID = validator.validated.id
+  const { stagingAddress } = validator.validated.data
+  const network = validator.validated.data.network || config.network
```

Comment:
We have a tendency to use lowercase network names in requests sometimes. It might be worth just uppercasing this string